src/main.qml

```diff
@@ -12,8 +16,12 @@ Window {
     height: 600
     visible: true
     title: qsTr("Remote Control")
-    color: Skin.darkGray
+    color: Skin.green
+
+    Keyboard {
+
+    }
 
     // Creation of the application's skeleton
-    ScoreSkeleton {}
+    //ScoreSkeleton {}
```

Comment:
I guess this means that this PR is a big WIP, no ? :p

Reply:
Yes... Sorry, I got confused in two PRs :(